examples/llm/inference/convert_composer_to_hf.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""TBD."""
+
+import json
+import tempfile
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Any, Dict, Optional, Union
+
+import sentencepiece as spm
+import torch
+from composer.utils import get_file, safe_torch_load
+from transformers import (AutoConfig, AutoTokenizer, PretrainedConfig,
+                          PreTrainedTokenizer)
+
+from examples.llm import MosaicGPTConfig
+
+
+def get_hf_config_from_composer_state_dict(
```

Comment:
leave a comment (and assign me a jira) to remove these functions (or make extremely minimal) after the next composer release.